Spring.jsx

```diff
@@ -386,7 +303,7 @@ export let TransitionSpring = React.createClass({
       let mergedVals = mergeDiffObj(
         currVals,
         endValue,
-        key => willLeave(key, endValue, currVals, currV),
+        key => willLeave(key, endValue, currVals, currV)
```

Comment:
Maybe it's because you didn't enable the comma transform in babel? It's es7 I think. Anyhow, the real library will ship the transpired version so we'll be fine